admin/InitDb.pl

```diff
@@ -49,8 +49,8 @@ sub RequireMinimumPostgreSQLVersion
 
     my $version = $sql->select_single_value(q{SELECT current_setting('server_version_num')});
 
-    if ($version < 12000) {
-        die 'MusicBrainz requires PostgreSQL 12 or later';
+    if ($version < 160000) {
```

Comment:
```suggestion
    if ($version < 16000) {
```

Reply:
Thanks, I intended to leave a comment here but forgot. The previous check was incorrect; they increased the length of the number when their versioning system changed with major release 10. Here you can see what numbers are returned for v12, 15, and 16 respectively:

```
psql (12.16 (Ubuntu 12.16-1.pgdg22.04+1))
Type "help" for help.

musicbrainz_db=> SELECT current_setting('server_version_num');
 current_setting
-----------------
 120016
(1 row)
```

```
psql (15.6 (Homebrew), server 15.5 (Homebrew))
Type "help" for help.

musicbrainz_db=> SELECT current_setting('server_version_num');
 current_setting
-----------------
 150005
(1 row)
```

```
psql (16.1 (Debian 16.1-1.pgdg120+1))
Type "help" for help.

musicbrainz_json_dump=> SELECT current_setting('server_version_num');
 current_setting
-----------------
 160001
(1 row)
```

> Surprised this didn't cause any issues with our tests, don't we test InitDb?

Yes, but we don't test this particular condition (nor multiple versions of PostgreSQL).